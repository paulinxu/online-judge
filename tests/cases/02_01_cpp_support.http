###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 407
# content-type: application/json
# date: Thu, 25 Jul 2024 15:33:05 GMT
# 
# {"cases":[{"id":0,"info":"","result":"Compilation Success","time":0},{"id":1,"info":"","result":"Accepted","time":0}],"created_time":"2024-07-25T15:33:05.585396906Z","id":0,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); }","user_id":0},"updated_time":"2024-07-25T15:33:05.798417051Z"}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"C","problem_id":0,"source_code":"#include <stdio.h>\nint main() { printf(\"Hello World!\\n\"); return 0; }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 436
# content-type: application/json
# date: Thu, 25 Jul 2024 15:33:05 GMT
# 
# {"cases":[{"id":0,"info":"","result":"Compilation Success","time":0},{"id":1,"info":"","result":"Accepted","time":0}],"created_time":"2024-07-25T15:33:05.800106720Z","id":1,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"C","problem_id":0,"source_code":"#include <stdio.h>\nint main() { printf(\"Hello World!\\n\"); return 0; }","user_id":0},"updated_time":"2024-07-25T15:33:05.944381527Z"}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"C++","problem_id":0,"source_code":"#include <iostream>\nint main() { std::cout<<\"Hello World!\"<<std::endl; return 0; }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 450
# content-type: application/json
# date: Thu, 25 Jul 2024 15:33:05 GMT
# 
# {"cases":[{"id":0,"info":"","result":"Compilation Success","time":0},{"id":1,"info":"","result":"Accepted","time":0}],"created_time":"2024-07-25T15:33:05.945794410Z","id":2,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"C++","problem_id":0,"source_code":"#include <iostream>\nint main() { std::cout<<\"Hello World!\"<<std::endl; return 0; }","user_id":0},"updated_time":"2024-07-25T15:33:06.237954992Z"}
