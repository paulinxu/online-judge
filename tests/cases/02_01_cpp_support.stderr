[2024-07-25T15:33:04Z INFO  actix_server::builder] starting 16 workers
[2024-07-25T15:33:04Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime
[2024-07-25T15:33:05Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET / HTTP/1.1" 404 0 "-" "-" 0.000270
[2024-07-25T15:33:05Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 407 "-" "-" 0.213474
[2024-07-25T15:33:05Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 436 "-" "-" 0.144542
[2024-07-25T15:33:06Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 450 "-" "-" 0.292452
